Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -62,7 +62,9 @@ void CGFontSetFTFontSize(id uiFont, void* ftFont, float pointSize, float scale)
 }
 
 CGSize CGFontDrawGlyphsToContext(CGContextRef ctx, WORD* glyphs, DWORD length, float x, float y) {
-    return ctx->Backing()->CGFontDrawGlyphsToContext(glyphs, length, x, y);
```

Comment:
Requires ongoing investigation.

---
In reply to: [83068337](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83068337) [](ancestors = 83068337,83026505)